packages/css/src/components/page-menu/page-menu.scss

```diff
@@ -15,6 +16,17 @@
   box-sizing: border-box;
 }
 
+@mixin reset-button {
```

Comment:
This might be a good reason the make a reusable mixin for buttons?

Reply:
It might. I’ll add a separate ticket for it.